### DISCUSSION TITLE
我是说啊，你们跑之前，给国内出点实质建议？

### DISCUSSION THREAD

**Discussion by suyeq** [The-Run-Philosophy-Organization/run#308] [General]
是实质建议，骂战和阴阳怪气就不必了，单独列个目录。我觉得嘛，跑出去后，我最后还是会回国内，叶落归根。

**Comment by Nagatosang** [comment]
给谁？ 国家？ 哈哈哈哈

**Comment by ghost** [comment]
> 是实质建议，骂战和阴阳怪气就不必了，单独列个目录。我觉得嘛，跑出去后，我最后还是会回国内，叶落归根。

如果中国有言论自由，我估计我都不会润（

**Reply by sseed5566** [reply]
润了你又能如何？还扯言论自由呢，你低级了

**Comment by suyeq** [comment]
我只是提个建议，没有让你们列出你们跑的理由。单纯的跑改变不了任何事情，我甚至觉得润学家不如效仿民国时期成立一个学派或者组织，集思广益或许还能改变点什么

**Reply by cambrian-rock** [reply]
附议（如果组织化可以国外NGO（但有涉及境外势力的危险），国内就别想了

**Reply by sychen12345** [reply]
不可能的，别想了，1989年之后，成立任何群体组织都会被严苛监控的，之前搞学联之后就是这样了。

**Reply by wedxz** [reply]
理想已死，江湖不再见

**Reply by yeebowang** [reply]
会被监控的，不可能用GITHUB通信

**Comment by ghost** [comment]
> 我只是提个建议，没有让你们列出你们跑的理由。单纯的跑改变不了任何事情，我甚至觉得润学家不如效仿民国时期成立一个学派或者组织，集思广益或许还能改变点什么

哦对了民国时期还能创立学派或组织，现在估计不行吧（

**Comment by hanayashiki** [comment]
国内最近几年发展的很不错，没啥好润的

**Reply by hanayashiki** [reply]
就是瞧不起咱中国人呗

**Comment by Evcklli** [comment]
当然是让总书记一直连任到底

**Comment by mitchellcuevas** [comment]
实质建议？有用吗。

**Comment by leon-fudan** [comment]
我觉得吧，如果你有落叶归根的想法，最好不要出去。活不过两集估计。

**Comment by sgfcz** [comment]
你不会觉得自己真的有萧何张良之才吧。。。还实质建议，都是普通人吹凉自己那碗粥就差不多得了

**Comment by HuarenYu** [comment]
建议？直接封号伺候

**Reply by sseed5566** [reply]
现在不封号了，你能做出个狗屁？刷盘子送外卖的时候当心点就行了，国家大事还真不是你这种人能决定的

**Comment by XifeiNi** [comment]
我们小心点吧 别github也被封了

**Reply by AntEffect** [reply]
说得好像小心了就不会被封一样，小白兔挨打是因为它戴帽子了吗？

**Comment by QAQ10028964** [comment]
支持共产党肏你们

**Comment by lance111l** [comment]
跑的人获得了其他国籍  早已忘祖了  别想他们的建议  
像韩国一样  禁止人境

**Comment by Silvertraces** [comment]
- replace the dictator and the dominant government. (Impossible)
- cease the increasing scarlet economy. (Impossible)
- be practical, let the elites guide the trend, not politics.  (Hard)

# to sum up: there's no way but a death end.

**Comment by jundaodao** [comment]
您的邮件我已经收到了，我会尽快回复您！                            ————焦江昆

**Comment by vulcan2333** [comment]
看你准备过来干什么？投资移民，工签，旅游签还是走线去美国。不管哪个渠道，钱和英语是需要的，还有就是吃苦的准备。如果不了解，先去北漂一年，就懂了

**Comment by sgfcz** [comment]
还真在北漂中。。。。。有些感触吧

发件人: vulcan2333 ***@***.***>
发送时间: 2023年8月1日 19:34
收件人: The-Run-Philosophy-Organization/run ***@***.***>
抄送: sgfcz ***@***.***>; Comment ***@***.***>
主题: Re: [The-Run-Philosophy-Organization/run] 我是说啊，你们跑之前，给国内出点实质建议？ (Discussion #308)


看你准备过来干什么？投资移民，工签，旅游签还是走线去美国。不管哪个渠道，钱和英语是需要的，还有就是吃苦的准备。如果不了解，先去北漂一年，就懂了

**Comment by vulcan2333** [comment]
又是哪儿来的傻逼高华，操你妈你家但凡不是常委，你在中国算个鸡吧啊，

**Comment by vulcan2333** [comment]
翻墙的傻逼就凭你也舔着逼脸能决定国家大事吗？另外臭傻逼自己觉得只能去送外卖刷盘子于是乎就觉得谁都是刷盘子送外卖吗？

**Reply by sseed5566** [reply]
你就是送外賣刷盤子的，你以為你是什麼高級東西。

**Comment by vulcan2333** [comment]
傻逼以己度人觉得别人只能刷盘子送外卖，真是可怜

**Reply by sseed5566** [reply]
那你是干啥的呀？臭傻逼，你润到民主自由的世界里，也没见你有什么建树呀？是体制的问题还是你脑子的问题呀

**Comment by mocuishle1998** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
